src/workspaces/features/DeleteWorkspaceDialog/DeleteWorkspaceDialog.tsx

```diff
@@ -0,0 +1,79 @@
+import Button from "core/components/Button";
+import Dialog from "core/components/Dialog";
+import Spinner from "core/components/Spinner";
+import useForm from "core/hooks/useForm";
+import { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { useRouter } from "next/router";
+import { useDeleteWorkspaceMutation } from "workspaces/graphql/mutations.generated";
+import { Workspace } from "graphql-types";
+import Router from "next/router";
+import useCacheKey from "core/hooks/useCacheKey";
+
+type DeleteWorkspaceDialogProps = {
+  onClose(): void;
+  open: boolean;
+  workspace: Omit<
```

Comment:
Better to reverse it like : `Pick<Workspace, "id">` WDYT?

Reply:
Yes, it's simpler